restaurant/src/app/admin/update-status.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient,HttpErrorResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UpdateStatusService {
+ 
+  
+  
+  constructor(private http:HttpClient) { }
+
+  update(){
+     var id = localStorage.getItem('update');
+    this.http.patch( `http://localhost:8080/admin/update?id=${id}`,{}).subscribe(data =>{
+      console.log(data);
```

Comment:
subscribe to this data in the component. and if an error happens, notify users via toastr